airflow/hatch.toml

```diff
@@ -0,0 +1,11 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["27", "38"]
+version = ["v1", "v2"]
+
+[envs.default.overrides]
+matrix.version.env-vars = "AIRFLOW_VERSION"
+
+[envs.latest.overrides]
+AIRFLOW_VERSION = "latest"
```

Comment:
@ofek I'm not sure this is the correct way to configure the `latest` env. Could you take a look?